core/os/android/adb/device.go

```diff
@@ -195,7 +195,7 @@ func scanDevices(ctx context.Context) error {
 
 	for serial, status := range parsed {
 		device, ok := cache[serial]
-		if !ok {
+		if !ok && status == bind.Status_Online {
```

Comment:
I'm a little bit on the fence about this. A few thoughts:

1. The package doesn't say anything about filtering to just online devices. It makes the status field particularly pointless. There's a few scattered comments that should be updated for this change in behavior.
1. I'm not sure if remote devices will always be 'Online' and not sometimes 'Unknown'.
1. Would it not be better to show 'Unauthorized' in the client?

For the sake of the smallest delta, I can approve this on the condition we revisit it. Alternatively, how do you feel about a `registry.RemoveDevice()` then `registry.AddDevice()` if the status changes?